internal/packagefmt/schemaextract/analyze_ts_test.go

```
@@ -0,0 +1,118 @@
+	capabilities, err := analyzeTS(context.Background(), source, "my.pkg", dir, fakeRunner)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if len(capabilities) != 1 || capabilities[0].ID != "my.pkg.hello" {
+		t.Fatalf("capabilities = %+v", capabilities)
+	}
+	if capabilities[0].Description != "hello 说你好。" {
+		t.Fatalf("description = %q", capabilities[0].Description)
+	}
+	var schema map[string]any
+	json.Unmarshal(capabilities[0].InputSchema, &schema)
```

golangci-lint (2.13.2)

[error] Error return value of `json.Unmarshal` is not checked

(errcheck)